data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -69,29 +64,19 @@
 	private Integer processId;
 	private Long instanceId;
 	private String defaultHiveDbName;
+	private LineageQuery lineageQuery;
 
 	//	private int nextId = 0;
 	private int subquerySeq = 0;
 	private PersistenceUnit persistenceUnit;
 
-	public static void main(String[] args) throws IOException, ParseException,
-			SemanticException, Exception {
+	public static void lineageMain(LineageQuery lineageQuery,String args1, String args2, String args3) throws Exception {
 		LineageMain lineageMain = new LineageMain();
-
-		if (args.length == 4) {
-			lineageMain.instanceId = Long.parseLong(args[3]);
-			lineageMain.processId = Integer.parseInt(args[2]);
-			lineageMain.defaultHiveDbName = args[1];
-			lineageMain.query = args[0];
-
-		} else {
-//			lineageMain.instanceId = LineageConstants.instanceId;
-//			lineageMain.processId = LineageConstants.processId;
-//			lineageMain.defaultHiveDbName = LineageConstants.defaultHiveDbName;
-//			lineageMain.query = LineageConstants.query;
-			System.out.println("Error: Invalid inputs for LineageMain");
-			throw new Exception("Invalid inputs for LineageMain");
-		}
+		lineageMain.instanceId = Long.parseLong(args3);
+		lineageMain.processId = Integer.parseInt(args2);
+		lineageMain.defaultHiveDbName = args1;
+		lineageMain.query = lineageQuery.getQueryString();
+		lineageMain.lineageQuery = lineageQuery;
 
 		lineageMain.getLineageInfo();
 //		lineageMain.generateOperatorTree(new HiveConf(), query);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)